provisioners/roles/postgres/tasks/main.yml

```diff
@@ -5,8 +5,28 @@
     dest=/etc/yum.repos.d/CentOS-Base.repo
   sudo: yes
 
+- name: install cert utilities
```

Comment:
So installing ca-certificates on a fresh vm and jumping straight to the "ensure PGDG Yum repo is installed" task works for you? I will try testing this when I reach a good stopping point.